servicetalk-http-api/src/test/java/io/servicetalk/http/api/DefaultHttpExecutionStrategyTest.java

```diff
@@ -161,8 +161,11 @@ void wrapServiceThatWasNotOffloaded(final Params params) throws Exception {
                 analyzer.instrumentedResponseForServer(svc.handle(ctx, req, ctx.streamingResponseFactory()))
                         .flatMapPublisher(StreamingHttpResponse::payloadBody)
                     .toFuture().get();
-            } catch (InterruptedException | ExecutionException e) {
-                fail("Unexepected exception", e);
+            } catch (InterruptedException e) {
```

Comment:
Consider changing `Runnable` to `Callable` instead of catching and rethrowing exceptions.